docs/contribution.rst

```diff
@@ -0,0 +1,125 @@
+======================
+Contributing to ecl2df
+======================
+
+Contributing to ecl2df is easiest on Linux computers. Windows has not been
+tested, and for Mac you will have to compile OPM yourself.
+
+Getting started as a developer
+------------------------------
+
+Clone the master repository from Github::
+
+  git clone https://github.com/equinor/ecl2df
+
+It is recommended that you also fork the repository on Github, in order to push
+your branches to your fork. After forking, add the fork to your cloned copy::
+
+  git remote add fork git@github.com:<youruser>/ecl2df
+
+This requires a valid login setup with SSH keys for you github account, needed
+for write access.
+
+After cloning, you should make a Python virtual environment in which you install
+ecl2df and its dependencies. If you want to create a new for ecl2df, you can
+do something like the following::
+
+  virtualenv venv-ecl2df
+  source venv-ecl2df/bin/activate
+
+and then run ``pip`` ::
+
+  pip install -e .[tests,docs]
+
```

Comment:
Recommend only pip install -e .

from the cloned directory (ecl2df/)

Reply:
This is developer instructions, and then I suppose it is appropriate to include tests and docs?

Added hints on current directory.